README.md

```diff
@@ -27,7 +27,11 @@ You can either run in Visual Studio, or using [Microsoft Tye](https://github.com
 - Rename `.env.example` file to `.env` file and provide PayPal ClientID and Secret.
 
 - Execute `run-tye.ps1`
-
+	- **Note**: If you meet with some shell issues regarding authorization. You may check with the following codes your access and then set your machine configuration. [for more detail](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/get-executionpolicy?view=powershell-7.2)
```

Comment:
`If you meet with some shell issues ...` we can express this better like: `If you come across shell issues ...`
We can use something like: `See Microsoft Powershell documentation` instead of `for more detail`.

Reply:
Thanks @gterdem  
I've fixed your suggestions.
FYI @enisn